apps/srt-tunnel.cpp

```diff
@@ -91,7 +91,7 @@ class Medium
     template <class DerivedMedium, class SocketType>
     static Medium* CreateAcceptor(DerivedMedium* self, const sockaddr_in& sa, SocketType sock, size_t chunk)
     {
-        string addr = SockaddrToString((sockaddr*)&sa);
+        string addr = SockaddrToString(sockaddr_any((sockaddr*)&sa, sizeof sa));
```

Comment:
This is a fix for the safer use of the `sockaddr_any` here (involves checking the source object size).